carsFilter/src/main/java/tkhal.filter.controller/MessageFilterController.java

```diff
@@ -32,11 +29,12 @@ public MessageFilterController() {
         carsFilter = new CarsFilter();
     }
 
+    /**
+     * This is the main method which getting cars, filtration them,
+     * and writing to the new Kafka topic
+     */
     public static void main(String[] args) {
-        /**
-         * This is the main method which getting cars, filtration them,
-         * and writing to the new Kafka topic
-         */
+
         new MessageFilterController();
```

Comment:
this is bad!
extract everything from `main` to another method
`main` should create instance of class and execute that method on that instance
apply for all mains

Reply:
done